### PR TITLE
Handle changesets with a single node, fixes #73

### LINKED ISSUE #73: Uncaught Error: Invalid LngLat object: (NaN, NaN) on some changesets
Getting a:

```
bundle.js:4350 Uncaught Error: Invalid LngLat object: (NaN, NaN)
```

on certain changesets. Not sure of the cause, but just dropping it here. Here's a link to reproduce https://osmlab.github.io/changeset-map/#42727347

cc @batpad @ajithranka

### PR DESCRIPTION
**Pull request by ajithranka**
If only a single node was modified in a changeset, `map.fitBounds` breaks because top-left and bottom-right point to the same lat/lng.

Fixed by adding a little padding to the bounds.

cc: @geohacker @batpad

### CODE REVIEW

**Comment by geohacker** [comment]
Nice catch @ajithranka. 🚢

File: dist/bundle.js [hunk 1]
```diff
@@ -667,9 +667,23 @@ function render(hash, options) {
             }
         });
 
+        // Special case: If a single node was changed, then
+        //    bbox.left == bbox.right, and
+        //    bbox.top == bbox.bottom
+        // In this case, add a little padding to avoid breaking fitBounds
+        var {left, right, top, bottom} = bbox
```
Comment (batpad) [inline dist/bundle.js]: Can we please semi colons please;

Comment (ajithranka) [inline dist/bundle.js]: It should work on most recent browsers. Better not to use destructuring, just in case.

**Comment by planemad** [comment]
Awesome! was wondering why https://osmcha.mapbox.com/42842703/ was not working

**Comment by ajithranka** [comment]
@planemad It should work after we merge this PR. Tested it locally.

**Pull request merged**